walkthroughs/howto-http-retries/README.md

```diff
@@ -0,0 +1,64 @@
+# HTTP Retry Policy example
+
+1. Clone this repository and navigate to the walkthrough/http-retry-policy folder, all commands will be ran from this location
+
```

Comment:
I should have asked for this in the header based routing one as well, but can we add an overview with a short blurb describing this walkthrough? The http2 one is pretty good.